components/Checkbox/Checkbox.jsx

```diff
@@ -8,13 +8,20 @@ import Icon from '../Icon';
 import CheckboxGroupContext from './CheckboxGroupContext';
 import { colors, spacing, baseFontSize } from '../shared/theme';
 
-const CHECKBOX_SIZE = '18px';
+const CHECKBOX_SIZE = '24px';
```

Comment:
What did you think about to take this value from `theme` ?

Reply:
I would keep it the way it is. Because this constant is used only here.
What do you think?

Reply:
Ok